fesvr/htif.h

```diff
@@ -93,7 +100,8 @@ class htif_t : public chunked_memif_t
   addr_t sig_len; // torture
   addr_t tohost_addr;
   addr_t fromhost_addr;
-  int exitcode;
+  // Set to a value by exit() when the simulation should exit.
+  std::optional<int> exitcode;
```

Comment:
Make this `uint64_t`; after all, it takes Spike less than a minute to execute enough instructions to overflow `int`.

Reply:
This is just the exit code written to htif, not the number of instructions. It was already `int`.

The number of instructions is `instruction_limit` below which is `unsigned long long`.

Reply:
Sorry, I read through the code too quickly.